.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
```suggestion
**Please describe the feature you'd like to see**
A clear and concise description of what problem feature solves or what use-case it would enable.
e.g. "I really wish I could ...", "I'm sad that I can't..."
```

Reply:
@dimberman it may be worth making a PR for the `astronomer-providers` project, so our feature issue has similar descriptions - I had copied these paragraphs from there:
https://github.com/astronomer/astronomer-providers/blob/main/.github/ISSUE_TEMPLATE/feature_request.md